src/utils/helpers.js

```diff
@@ -16,6 +16,22 @@
 import commonOptions from '../utils/options';
 import { printResult } from '../utils/print';
 
+const regExpAddress = /^\d{1,21}[L|l]$/;
+const regExpAmount = /^\d+(\.\d{1,8})?$/;
+
+export const verifyAddress = (address) => {
+	if (!address.match(regExpAddress)) {
+		throw new Error(`${address} is not a valid address.`);
+	}
+	return true;
+};
+export const verifyAmount = (amount) => {
```

Comment:
Sorry, I know it was my suggestion, but these should really be `validateX`, not `verifyX`